.github/workflows/release.yml

```diff
@@ -59,10 +71,12 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
           repository: seiso/easy_infra
-          short_description: |
-            A docker container to simplify the use of Infrastructure as Code (IaC) tools
+          short-description: |
```

Comment:
This was a bug that I just happened to notice

Reply:
Tiny bug!